app/assets/stylesheets/application.css.scss

```diff
@@ -14,8 +14,7 @@
 
 @import 'base/*',
         'utilities/*',
-        'components/*';
-
-@import 'modules/*';
+        'components/*',
+        'modules/*';
```

Comment:
Line should be indented 0 spaces, but was indented 8 spaces<br>Prefer double-quoted strings